tools/oeedger8r/Emitter.ml

```diff
@@ -493,11 +493,6 @@ let gen_enclave_code (ec : enclave_content) (ep : edger8r_params) =
         printf
           "Warning: Function '%s': Reentrant ocalls are not supported by Open \
            Enclave. Allow list ignored.\n"
-          f.uf_fdecl.fname ;
-      if f.uf_is_switchless && not ep.experimental then
-        failwithf
-          "Function '%s': switchless ecalls and ocalls are not yet supported \
-           by Open Enclave SDK."
           f.uf_fdecl.fname )
```

Comment:
Why don't I see any other edger8r changes...is there nothing to change in the generated code for supporting switchless? That would be...unexpected (to me anyway).

Reply:
Most edger8r changes are already merged in this PR: https://github.com/openenclave/openenclave/pull/2031/files. Thanks to Anand.

Reply:
Oh yeah I remember that now :joy: